File: QuanLyVatTu_ASP/fix_lines.py
import sys
f = r'd:\A\QuanLyVatTu_ASP.NET\QuanLyVatTu_ASP\Views\SanPham\ProductDetail.cshtml'
with open(f, 'r', encoding='utf-8') as fp:
    lines = fp.readlines()
print(f"Total lines: {len(lines)}")
# Keep lines 1-906 (index 0-905) and lines 954+ (index 953+)
# Remove lines 907-953 (index 906-952)
keep = lines[:906] + lines[953:]
print(f"New total: {len(keep)}")
with open(f, 'w', encoding='utf-8') as fp:
    fp.writelines(keep)
print("Done!")
sys.exit(0)

File: QuanLyVatTu_ASP/fix_lines.js
const fs = require('fs');
const f = 'd:\\A\\QuanLyVatTu_ASP.NET\\QuanLyVatTu_ASP\\Views\\SanPham\\ProductDetail.cshtml';
const lines = fs.readFileSync(f, 'utf8').split('\n');
console.log('Total lines:', lines.length);
// Remove lines 907-953 (0-indexed: 906-952)
const keep = [...lines.slice(0, 906), ...lines.slice(953)];
console.log('New total:', keep.length);
fs.writeFileSync(f, keep.join('\n'), 'utf8');
console.log('Done!');
